pages.hi/common/git-init.md

```diff
@@ -0,0 +1,11 @@
+# git init
+
+> एक नया स्थानीय गिट रिपॉजिटरी शुरू करता है। अधिक जानकारी: https://git-scm.com/docs/git-init ।
+
+- एक नया स्थानीय भंडार शुरू करें:
+
+`git init`
+
+- एक नंगे पत्थरों के भंडार को शुरू करें, जो ssh के रिमोट के रूप में उपयोग के लिए उपयुक्त है:
```

Comment:
Is this the correct translate for: `Initialize a barebones repository, suitable for use as a remote over ssh:`?
![Screenshot_20201009_111417](https://user-images.githubusercontent.com/1536740/95593937-ab0e3b80-0a20-11eb-95ce-8de314199fc1.png)

Reply:
updated

Reply:
I see the same text, is the Hindi text ok?

Reply:
@Athi223 I see the same text, is the Hindi text ok?

Reply:
Pushed the final changes, previous push probably failed due to network issues, my apologies.